mail-client/mutt/mutt-1.6.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Small but very powerful text-based mail client"
+DESCRIPTION="A text-based email client for Unix-like systems. \
+It was originally written by Michael Elkins in 1995 and released \
+under the GNU General Public License version 2 or any later version.
+
+The Mutt slogan is All mail clients suck. This one just sucks less."
+HOMEPAGE="http://www.mutt.org"
```

Comment:
IMHO it would be better to have
```
HOMEPAGE="http://www.mutt.org/"
```
I know most browsers hide the trailing slash when there is nothing after it, but, to be correct, a URL needs either a path or a trailing slash.
I also know that this is a detail and that some recipes are missing that trailing slash.
But I think we should have correct URLs in our recipes.
A practical reason for keeping the slash is that when wrong URLs are copy/pasted to forums that make hyperlinks with them, these hyperlinks do not always work because the wrapper that creates the hyperlinks is not able to correctly detect the end of the domain name when it is followed by a dot. I've already seen that in some places. This never happens with correct URLs ;-)